kalite/updates/static/js/updates/update_videos.js

```diff
@@ -167,6 +167,7 @@ $(function() {
         numVideos = youtube_ids.length;
 
         // Do the request
+        show_message("warning", "Wait for the video download to start.");
```

Comment:
Can you have this shown only when the user is using OS X? Also, please add a `gettext` call.